package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "scripts": {
     "start": "cross-env NODE_ENV=development webpack-dev-server --hot --inline --watch --content-base dist --port 8085",
-    "dist": "cross-env NODE_ENV=production webpack -p --config webpack.production.config.js",
+    "dist": "cross-env NODE_ENV=production webpack --config webpack.production.config.js",
```

Comment:
I've just tried OccurenceOrderPlugin, it shaves 1KB more off the bundle size... not much, but doesn't hurt to add it :)

I've removed the ```-p``` option because I find it better to have the equivalent options detailed in the config file, it's more explicit this way and it allows us to fine-tune the minification process.